### PR TITLE
Pull in the class object for the single class page view

### PR DESCRIPTION
**Pull request by mlamina**
This PR implements #9 and expands the JSON for the classes model returned by the Rails app and adjusts the new React component to look like the original classes list, **except** for the *actions* column.

### CODE REVIEW

File: client/app/bundles/Attendance/components/ClassScreen/ClassScreen.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React, { PropTypes } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
```
Comment (hex337) [inline client/app/bundles/Attendance/components/ClassScreen/ClassScreen.jsx]: I just recently saw that they moved PropTypes out. I think, in the future, if we're going to change a few instances, I'd rather keep things consistent and do them the old way, or replace all instances of it. For now, this is fine, I'll make sure I update the rest as I poke around more.

File: app/controllers/meetings_controller.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@ def index
     @meetings = Meeting.for_school(@_current_school).paginate(page: params[:page], per_page: 50)
     respond_to do |format|
       format.html { render :index }
-      format.json { render json: @meetings.to_json }
+      format.json { render json: @meetings.to_json({:include => :meeting_type, :methods => :member_count}) }
```
[the remark this answers is not in the document]
Comment (mlamina) [reply app/controllers/meetings_controller.rb]: Neither did I haha

File: client/app/bundles/Attendance/components/ClassScreen/ClassScreen.jsx (hunk 1 above)
Comment (mlamina) [reply client/app/bundles/Attendance/components/ClassScreen/ClassScreen.jsx]: I know I had the same thought. Didn't wanna do it in this branch, but we should do it eventually

**Comment by mlamina** [comment]
Sure, I'll add something to the README!

**Comment by mlamina** [comment]
Done! I don't know how to execute the rails tests yet, so that's TBD

**Comment by hex337** [comment]
Looks good :) I can add comments for the rails tests.

**Pull request merged**